backend/api/neuron/models/neuron.settings.json

```diff
@@ -25,14 +22,34 @@
       "required": false,
       "pluginOptions": {}
     },
-    "developmental_stages": {
-      "collection": "developmental-stage"
-    },
     "metadata": {
       "type": "text"
     },
     "wormatlas": {
       "type": "string"
+    },
+    "uid": {
+      "type": "uid"
+    },
+    "timepoints": {
+      "type": "json"
+    },
+    "embryonic": {
+      "type": "boolean"
+    },
+    "lineage": {
+      "type": "string"
+    },
+    "location": {
+      "type": "string"
+    },
+    "synapse": {
```

Comment:
we don't want to have `synapse` entry in neuron, I think Strapi offers another one-to-many relationship that won't add an entry to the many-side of the relationship.

Reply:
A didn't know strapi supported that. Cool! 
Seems to be right now
![image](https://user-images.githubusercontent.com/19196034/126817191-d206ac30-a251-45e4-8839-478b3ea20161.png)
![image](https://user-images.githubusercontent.com/19196034/126817227-b9f83c0b-f85a-4ce7-b787-b8916a832fc9.png)
![image](https://user-images.githubusercontent.com/19196034/126817248-2b34e79e-6b22-4435-a2f5-311923754444.png)